package.json

```diff
@@ -11,15 +11,17 @@
     "plugin"
   ],
   "scripts": {
-    "test": "$(yarn bin)/mocha"
+    "test": "mocha"
   },
   "devDependencies": {
     "chai": "4.2.0",
-    "fs-extra": "7.0.1",
-    "mocha": "5.2.0",
-    "webpack": "3.12.0"
+    "eslint": "^6.8.0",
```

Comment:
I don't prefer to have ESLint for such a small project to avoid having an additional dependency.
Happy to have `.prettierrc` or `.editorconfig` if the consistent coding style helps contributors.

Reply:
great point, pushed an updated commit